op-conductor/conductor/service.go

```diff
@@ -559,7 +559,9 @@ func (oc *OpConductor) startSequencer() error {
 
 		if uint64(unsafeInCons.BlockNumber)-unsafeInNode.NumberU64() == 1 {
 			// tries to post the unsafe head to op-node when head is only 1 block behind (most likely due to gossip delay)
-			if err = oc.ctrl.PostUnsafePayload(context.Background(), unsafeInCons); err != nil {
+			// TODO:
```

Comment:
?

Reply:
related to the below breaking change comment to `PostUnsafePayload`. Will update the comment to refer to a ticket. Was unsure whether or not we were going to address it as part of this PR. But since that thing isn't in prod yet, we shouldn't block.

Reply:
I'd expect this to not pass CI b/c of the dangling todo